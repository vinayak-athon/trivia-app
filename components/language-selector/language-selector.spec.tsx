import React from "react";
import { render, cleanup } from "@Test/utils";
import { LanguageSelector } from "@Components";

afterAll(cleanup);

describe("Components", () => {
  describe("LanguageSelector lang=en", () => {
    const { container } = render(<LanguageSelector />);
    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
});
