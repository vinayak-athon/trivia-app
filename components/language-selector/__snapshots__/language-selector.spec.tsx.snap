// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components LanguageSelector lang=en should match snapshot 1`] = `
.c0 {
  cursor: pointer;
  color: #000000;
  background: #f9da2e;
  border-radius: 5px;
  padding: 5px 10px;
  font-family: Arial;
  font-weight: 700;
  font-size: 25px;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  text-transform: uppercase;
}

.c1 {
  background-color: green;
  color: inherit;
  font-size: 10px;
  border: none;
}

.c2 {
  background-color: inherit;
  color: inherit;
  font-size: 10px;
  border: none;
}

<div>
  <button
    class="c0 c1"
  >
    English
  </button>
  <button
    class="c0 c2"
  >
    Spanish
  </button>
</div>
`;
